src/nvim/os/os_defs.h

```diff
@@ -18,11 +18,15 @@
 #endif
 
 #if defined(UFS_MAXNAMLEN) && !defined(MAXNAMLEN)
-# define MAXNAMLEN UFS_MAXNAMLEN    /* for dynix/ptx */
+# define MAXNAMLEN UFS_MAXNAMLEN    // for dynix/ptx
```

Comment:
Must use `/*` comments inside macros (and lint should allow this, or ignore lint in that case).

Note that inline comments are still preferred for concise descriptions like this, so I do not suggest moving the comment to its own line.